svelte/src/scripts/draw/drag-and-drop.ts

```diff
@@ -33,7 +36,32 @@ export function addDragAndDrop(
 				// Calculate node movement, keeping the transformation (specifically the scaling) in mind.
 				const xt = node.x! + source.movementX * (1 / (drawSettings.transformation?.k ?? 1));
 				const yt = node.y! + source.movementY * (1 / (drawSettings.transformation?.k ?? 1));
+				if (drawSettings.isPanning) {
```

Comment:
i think this part is ruining the performance on large enough data. at least on my machine @Adityapnn811 mind testing on your?